SmartPlaces.Facilities/README.md

```diff
@@ -1,27 +1,70 @@
 # Smart Facilities
 
-This folder contains the code, configuration, and documentation for the Smart Facilities Components. This includes several libraries that are created as Nuget Packages, and some sample code.
+This folder contains the code, configuration, and documentation for the SmartPlaces.Facilities Components. This includes several libraries that are also packaged as Nuget Packages, and some sample code.
 
-### Required Software
+## Required Software
 
-1. Visual Studio 2022+
-2. Visual Studio Code
+1. Visual Studio 2022+ or Visual Studio Code
 
-## Getting Started
+## Libraries
 
-These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+There are two core libraries which have been extended to work with Mapped as a source provider. Other twin source providers are also encouraged to accelerate customer scenarios by partnering through Microsoft into RealEstateCore and other output ontologies built on Azure Digital Twins.
 
-1. Clone the repo
-3. Compile the Solution.
+| Name | Description | Read More |
+| --- | --- | --- |
+| OntologyManager | Defines base classes and helpers which allow mapping from one ontology to another | [Read More](./lib/OntologyMapper/README.md) |
+| OntologyManager.Mapped | Defines the mappings from the Mapped ontology to Rec and Willow ontologies | [Read More](./lib/OntologyMapper.Mapped/README.md) |
```

Comment:
We are REC and Willow, tomorrow we could have Elastacloud or Bosch, I am wondering if we should formulate it in such way to be more generic.

"Defines the mappings from the Mapped ontology to target ontologies, like REC, or Willow"